### PR TITLE
Update smart AHU

### PR DESCRIPTION
**Pull request by Karthi8883**
Updated smart AHU

### CODE REVIEW

**Comment by tasodorff** [comment]
validator errors

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 1]
```diff
@@ -870,6 +870,21 @@ AHU_DFSS_DXZC_ECOND_ZHM:
   - ECOND
   - ZHM
 
+AHU_SS_CHWDT_STM_RTM_RWISOVPC:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:873]: incorrect use of SS (should be SFSS) and mismatching of terms like discharge and supply; no mention of outisde air temperature control, requisite for modeling any AHU. you also use an optional return air temperature setpoint, which should exist as a required field on an abstract type.

please reconstruct.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 1 above)
Comment (Karthi8883) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: Updated the model

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 2]
```diff
@@ -870,6 +870,22 @@ AHU_DFSS_DXZC_ECOND_ZHM:
...
   - ZHM
 
+AHU_DFSS_CHWDT_STM_RTC_RWISOVPC_OAFC:
+  description: "Smart AHU."
+  is_canonical: true
+  implements:
+  - AHU
+  - DFSS
+  - CHWDT
+  - STM
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:880]: DTM, not STM. you use DFSS and CHWDT above.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml (hunk 2 above)
Comment (Karthi8883) [reply ontology/yaml/resources/HVAC/entity_types/AHU.yaml]: Modified to DTM

**Review by tasodorff — APPROVED** [review]

**Comment by mschulze17** [comment]
@cstirdivant yours to merge

**Pull request merged**